### ISSUE TITLE
Check that `git init` is clearly defined and formatted

### ISSUE THREAD

**Issue by brianamarie** [github/training-kit#753]
## Overview
**TL;DR**
As brought up in https://github.com/github/training-kit/pull/752, `git init` should be clearly defined and formatted to not be confused with other commands, especially those involving remotes.

#### Screenshots
![图片](https://user-images.githubusercontent.com/24759802/84172848-0cdc8a00-aaaf-11ea-9b48-9656aaf36eee.png)

### How to Contribute
- [ ] Comment on this issue if you'd like to work/collaborate on it!
- [ ] Fork the [repository](https://github.com/github/training-kit).
- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://services.github.com/on-demand/intro-to-github/) is a refresher course if you're unsure about how to make a change on GitHub.
- [ ] Push your changes to your repository.
- [ ] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/`.

### Questions?
- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!

**Comment by anneloreegger** [comment]
I'd like to work on this issue. Is it still available?

**Comment by brianamarie** [comment]
Hi @anneloreegger! Yes, this is still available! 😊  I look forward to seeing your pull request. Please let me know if you have any questions!
